test/util/test_path_provider.py

```diff
@@ -0,0 +1,173 @@
+import os
+from pathlib import Path
+from abc import ABC, abstractmethod
+
+
+class TestPathProviderBase(ABC):
+    """
+        used to generate the files path's in all tests
```

Comment:
couple of notes here. Can you align the whole paragraph to have the same line breaks. So that it becomes
"""
....
...
...
"""
and not
"""
....
........
..
"""

Reply:
At the beginning of the sentence capitalize every letter